packages/jellyfish-testing/src/testingwrapper.ts

```diff
@@ -0,0 +1,34 @@
+import { RegTestFoundationKeys } from '@defichain/jellyfish-network'
+import { ContainerGroup, DeFiDContainer, MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing, TestingGroup } from '.'
+
+type InitDeFiContainerFn = (index: number) => DeFiDContainer
+function defaultInitDeFiContainer (index: number): DeFiDContainer {
+  return new MasterNodeRegTestContainer(RegTestFoundationKeys[index])
+}
+export class TestingWrapper {
```

Comment:
Perhaps it would make sense to make all the methods static? Since you are not holding on to any state for TestingWrapper? No point init TestingWrapper once?

Either way?
1. `export const TestingWrapper = {}`
2. `static create ()...`

Reply:
> 

yea agreed. but i tried the last time to overload a static function but it doesnt seem to be possible. hence no choice, im forced to use non static

Reply:
@chee-chyuan is this referring to?
![issue](https://user-images.githubusercontent.com/1998130/154204416-2cd2d232-9b4d-46dd-a640-b6f1e3300c34.png)

Reply:
@kodemon gave me a workaround. will apply that one and push